client/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
This currently says "All bots are free to visit server" which is probably fine since malicious bots will ignore robots.txt anyway.